src/Element/Element.php

```diff
@@ -204,6 +196,6 @@ public function getOuterHtml()
      */
     protected function elementNotFound($type, $selector = null, $locator = null)
```

Comment:
agreed. Precisely my point.
there was no logic before, but it allowed to access the session without using the deprecated getter